src/main/steps/c100-rebuild/miam/mediator-document/content.ts

```diff
@@ -0,0 +1,73 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+const en = () => ({
+  headingTitle: 'Do you have a document signed by the mediator?  ',
+  docSigned:
+    'The mediator should give you a signed document to confirm you attended a MIAM, or do not need to attend. If you do not have a document, you should ask the mediator for one.',
+  one: 'Yes',
+  two: 'No',
+  errors: {
+    haveDocSigned: {
+      required: 'Select yes if you have a document signed by the mediator',
+    },
+  },
+});
+
+const cy = () => ({
+  headingTitle: 'Do you have a document signed by the mediator? - welsh  ',
```

Comment:
@monalisasarkar1 Can we rename 'headingTitle' to 'title' ?

Reply:
done